webapp/pages/profile/_id/_slug.vue

```diff
@@ -327,6 +334,11 @@ export default {
     },
   },
   methods: {
+    handleTab(tab) {
+      this.tabActive = tab
+      this.Post = null
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-06T23:42:53Z" title="Friday, June 7th 2019, 1:42:53 am +02:00">Jun 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -317,6 +328,11 @@ export default {
     },
   },
   methods: {
+    handleTab(tab) {
+      this.tabActive = tab
+      this.Post = null
```


This is probably not very intuitive: Setting the `Post` to null ensures that you don't see two loading spinners at the same time. (We have two for different purposes, because we might switch tabs or we fetch more contributions).

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-06-07T04:56:15Z" title="Friday, June 7th 2019, 6:56:15 am +02:00">Jun 7, 2019</time>_

---


ok, the exact dependencies are not quite familiar to me yet. because the first time you call up a profile / your own profile, the first TAB is displayed. and i had the problem with the dynamic classes. 
Case: if I set an active CLASS statically to the postTAB then it is strangely double there. 
which can also be a bug in the devconsole in chrome. however it deletes the CLASS active cleanly. but with postTAB it is set twice! 
which should not be allowed. but dynamic CLASS behave differently than addClass. :class is apparently very stupid. so in the sense that it works like a hammer. but I got that out soon